src/OrchardCore.Modules/OrchardCore.ContentLocalization/Views/ContentCulturePickerContainer.cshtml

```diff
@@ -1,14 +1,19 @@
-@using System.Globalization
-
+@{ 
+    var otherCultures = ((IEnumerable<CultureViewModel>)Model.SupportedCultures).Where(entry => !string.Equals(entry.Name, Model.CurrentCulture.Name, StringComparison.OrdinalIgnoreCase));
+}
 <ul>
     <li class="nav-item dropdown">
         <a class="nav-link dropdown-toggle" href="#" id="oc-culture-picker" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">@Model.CurrentCulture.DisplayName</a>
         <div class="dropdown-menu" aria-labelledby="oc-culture-picker">
-            @foreach (var culture in ((IEnumerable<CultureInfo>)Model.SupportedCultures).Where(entry => !string.Equals(entry.Name, Model.CurrentCulture.Name, StringComparison.OrdinalIgnoreCase)))
+            @foreach (var culture in otherCultures)
             {
-                @await DisplayAsync(await New.ContentCulturePickerLink(
-                   Culture: culture
-                ))
+                <a asp-action="RedirectToLocalizedContent"
```

Comment:
Why you didn't use shape here?

Reply:
> I am ok with either but people will most likely always override the container and want to set a custom css class on the link.

That's what I quote from you when I started working on culture picker :), I'm fine with simplicity